src/IconButton/index.jsx

```diff
@@ -75,6 +76,7 @@ IconButton.propTypes = {
   onClick: PropTypes.func.isRequired,
   /** Type of button (uses Bootstrap options) */
   variant: PropTypes.oneOf(['primary', 'secondary', 'success', 'warning', 'danger', 'light', 'dark']),
+  size: PropTypes.oneOf(['sm', 'inline']),
```

Comment:
It looks like you've got a default size of 'md', but that's not a valid type for the `size` prop?

Reply:
Yeah I wasn't sure if I should surface the md size or not because it's technically the default.

Class name `btn-icon-md` doesn't actually do anything in the css, but I felt it was weird for the default to be `btn-icon-` (just empty).

I don't know if I should surface the `md` size, or change the default to be empty and just not append the size class name if one wasn't provided?

Reply:
I'm just going to surface the `md` as a size variant unless anyone has an objection